src/solver/RootFinding.jl

```diff
@@ -0,0 +1,22 @@
+# module RootFinding
+
+include("coeffs.jl")
+include("nemo_stuff.jl")
+include("solve_helpers.jl")
+include("postprocess.jl")
+include("univar.jl")
+include("ia_helpers.jl")
+include("polynomialization.jl")
+include("attract.jl")
+include("ia_main.jl")
+include("main.jl")
+
```

Comment:
All includes should be at the top level

Reply:
can you check the current includes? is this what you mean?